task/fbc-related-image-check/0.1/fbc-related-image-check.yaml

```diff
@@ -48,11 +52,13 @@ spec:
           fi
         done
         if [ -z "$FAILEDIMAGES" ]; then
-          HACBS_TEST_OUTPUT="$(make_result_json -r SUCCESS -s 1)"
+          note="Task \`$(context.task.name)\` succeeded: For details, check the Tekton task result \`HACBS_TEST_OUTPUT\`."
+          HACBS_TEST_OUTPUT=$(make_result_json -r SUCCESS -s 1 -t "$note")
           echo "${HACBS_TEST_OUTPUT}" | tee "$(results.HACBS_TEST_OUTPUT.path)"
         else
           echo "These images failed inspection: $FAILEDIMAGES"
-          HACBS_TEST_OUTPUT="$(make_result_json -r FAILURE -f 1)"
+          note="Task \`$(context.task.name)\` failed: Command \`skopeo inspect\` could not inspect images. For details, check the Tekton task log."
```

Comment:
Iike this type of `the`, should we also remove it? @jseseCCS

Reply:
i like how it reads better with "the," but i think you're right. for consistency, we should delete all of those. you think so too @hongweiliu17 ?